warcry_proj/warcry_proj/routes.py

```diff
@@ -0,0 +1,11 @@
+def includeme(config):
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+
+    # relevant routes for the Restful API
```

Comment:
+1, I'd review the API standards for the project that were sent over (and we can post more generally for OSS use in the readme later)